cpp/include/raft/core/operators.hpp

```diff
@@ -40,6 +40,14 @@ struct identity_op {
   }
 };
 
+struct no_op {
+  template <typename Type, typename... UnusedArgs>
+  constexpr RAFT_INLINE_FUNCTION void operator()(const Type& in, UnusedArgs...) const
+  {
+    return;
+  }
```

Comment:
Perhaps, we could remove the first argument and the return statement here?

Reply:
Oops, never mind, @Nyrio was faster with the comments :)

Reply:
Thanks! I removed the first argument, but kept the return for symmetry with all the other operators in the file :)